### DISCUSSION TITLE
Cannot close "Marking answers" hint

### DISCUSSION THREAD

**Discussion by gh640** [community/community#5158] [Discussions]
I tried creating a discussion thread in my repository. I clicked "Got it!" in "Marking answers" block but the block didn't close. Is this an expected behavior? I'd like to know how to close the block.

screenshot:

![Screen Shot 2021-08-18 at 13 44 55](https://user-images.githubusercontent.com/3532528/129838601-ab037261-0791-4151-a89a-42d4eee6e913.png)

screen recording:

https://user-images.githubusercontent.com/3532528/129838630-5b191c61-127d-437f-8595-14f04c3f1738.mov

https://github.com/gh640/discussions-sample-ja/discussions/2

I created the thread and answered it by myself. Maybe it's related.

Related: https://github.com/github/feedback/discussions/2903

**Comment by gh640** [comment]
"Mark as answer" button didn't work, either.

![Screen Shot 2021-08-18 at 13 52 08](https://user-images.githubusercontent.com/3532528/129838961-4040a509-c3c1-4ee2-a8ca-d231d841d249.png)

**Comment by j-f1** [comment]
I’m seeing this too. It looks like GitHub is sending back a 422 error from an API request triggered by clicking the button.

**Comment by grahamperrin** [comment]
Cross-reference: https://github.com/github/feedback/discussions/5249

**Comment by qwel-exe** [comment]
> I’m seeing this too. It looks like GitHub is sending back a 422 error from an API request triggered by clicking the button.

try inspecting the page and go to console if you get an error like  this 
![image](https://user-images.githubusercontent.com/72848513/131301168-fb3160b0-5eed-4d2c-87f8-bfccef3b1494.png)

then click  on the given link it will take you to a website 
you might get something like this 
![image](https://user-images.githubusercontent.com/72848513/131301417-877b05c9-b7bf-405b-9fc0-90eeb702173d.png)
Click on advance and click on proceed anyway  and then try to click on Got it mostly this will work if it does not then idk 

if it takes you to the networking tab then copy the link given  in the console and past it in your browser and do the top steps and then try

**Reply by j-f1** [reply]
That looks like a completely different issue.

**Reply by qwel-exe** [reply]
oh that i am using twitch to explain how to do

**Reply by qwel-exe** [reply]
because my comp tells  twitch is unsafe

**Reply by qwel-exe** [reply]
most of the time the API link will be unsafe and Github wont be able to call it form our computer
